posix/basic-unixint.lisp

```diff
@@ -312,3 +318,22 @@
   (atime   "st_atime"   :type time)
   (mtime   "st_mtime"   :type time)
   (ctime   "st_ctime"   :type time))
+
+#+darwin
+(cstruct stat "struct stat"
```

Comment:
Put the extra fields in the definition of struct stat and export the extra fields, see the examples in package.lisp